File: backend/src/main/java/com/instantmechanic/mechanic_api/model/ServiceRequest.java
package com.instantmechanic.mechanic_api.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ServiceRequest {

    private Long id;
    private String customerName;
    private String phoneNumber;
    private String vehicleNumber;
    private String service;
    private String problemDescription;
    private String mechanicName;
    private String status;
    private String requestTime;
    private LocalDateTime createdAt;
}

File: app/src/main/java/com/example/instantmechanic/data/model/ServiceRequest.kt
package com.example.instantmechanic.data.model

data class ServiceRequest(
    val customerName: String,
    val phoneNumber: String,
    val vehicleNumber: String,
    val service: String,
    val problemDescription: String,
    val id: Long = System.currentTimeMillis(),
    val mechanicName: String = "Apex Auto Care",
    val status: String = "Assigned",
    val requestTime: String = "Today, Just now"
)